src/index.html

```diff
@@ -1,12 +1,229 @@
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>Mayo Catalog</title>
+    <link rel="stylesheet" href="./styles/main.scss" >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header>
+        <div class="header">
+          <div class="countainer">
+            <div class="menu">
+              <div class="menu__logo">
+                <a href="#"><img src="./images/logo.png" alt="Logo" ></a>
+              </div>
+              <div class="menu__category">
+                <ul class="menu__category-list">
+                  <li class="menu__category-item">
+                    <a href="#" class="menu__category-link is-active"> Apple</a>
+                  </li>
+                  <li class="menu__category-item">
+                    <a href="#" class="menu__category-link"> Samsung</a>
+                  </li>
+                  <li class="menu__category-item">
+                    <a href="#" class="menu__category-link"> Smartphones</a>
+                  </li>
+                  <li class="menu__category-item">
+                    <a href="#" class="menu__category-link">
+                      Laptops & Computers
+                    </a>
+                  </li>
+                  <li class="menu__category-item">
+                    <a href="#" class="menu__category-link"> Gadgets</a>
+                  </li>
+                  <li class="menu__category-item">
+                    <a href="#" class="menu__category-link"> Tablets</a>
+                  </li>
+                  <li class="menu__category-item">
+                    <a href="#" class="menu__category-link"> Photo</a>
+                  </li>
+                  <li class="menu__category-item">
+                    <a href="#" class="menu__category-link"> Video</a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </header>
+
+      <main>
+        <div class="countainer">
+          <div class="catalog">
+            <div class="catalog__card">
+              <img src="./images/imac.jpeg" alt="imac" >
+              <h2>APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+              <h4>Product code: 195434</h4>
+              <div class="catalog__rating">
+                <div class="catalog__stars stars-4">
+                  <div class="stars-star"></div>
+                  <div class="stars-star"></div>
+                  <div class="stars-star"></div>
+                  <div class="stars-star"></div>
+                  <div class="stars-star"></div>
+                </div>
+
+                <h4>Reviews: 5</h4>
+              </div>
+              <div class="catalog__price">
+                <h3>Price:</h3>
+                <h1>$2,199</h1>
+              </div>
+              <button type="button">BUY</button>
```

Comment:
Here you should use a tag with appropriate data attribute. Also make separate class for adding style to it.

```suggestion
              <a data-qa="card-hover">BUY</a>
```